packages/gestalt/src/__snapshots__/Tabs.test.js.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`<Tabs /> matches snapshot with default props 1`] = `
 <div
-  className="box"
+  className="box marginEnd0 marginStart0"
```

Comment:
TIL looks like this is happening because we use `<Row />` in `<Tabs />`. (no changes required)

Reply:
Yeah, left comment in changes.